backend/migration_service.py

```
@@ -1,950 +1,266 @@
+                                label_filters: Optional[Dict[str, str]] = None,
+                                source_host: Optional[str] = None,
+                                source_ssh_user: str = "root",
+                                source_ssh_port: int = 22) -> ContainerDiscoveryResult:
+        """Discover containers for migration"""
+        return await self.discovery_service.discover_containers(
+            container_identifier, identifier_type, label_filters,
+            source_host, source_ssh_user
+        )
+    
+    async def analyze_containers_for_migration(self,
+                                             container_identifier: str,
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)